File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao08.py
maior = -1000
menor = 1000
while True:
    valor = input("\nSe deseja parar digite 'parar'\nDigite um número entre 0 e 1000: ")
    
    if (valor == "parar"):

        break
    else:
        numero = int(valor)
        if (numero > maior):
            maior = numero
        else:
            if (numero < menor):
                menor = numero
            
print(f"O maior número é {maior} e o menor número é {menor}")

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Java_Script/questao08.js
let maior = -1000
let menor = 1000
while (true) {
    let valor = prompt("Digite um número entre 0 e 1000: ")
    
    if (valor == "parar") {
        break
    }else{
        var numero = parseInt(valor)
        if (numero > maior){
            maior = numero
        }else {
            if (numero < menor) {
                menor = numero
            }
        }
    }
}
console.log(`O maior número é ${maior} e o menor número é ${menor}`)